src/components/AddNewBook.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const AddNewBook = () => (
+  <div className="AddNewBook">
+    <input type="text" placeholder="Book title" />
+    <input type="text" placeholder="Author" />
+  </div>
```

Comment:
- [ ] the form should have a submit button as well and it should also be wrapped in a form tag for it to work.